lib/puppet/provider/f5_command/rest.rb

```diff
@@ -35,7 +31,7 @@ def exists?
   end
 
   def create
```

Comment:
It seems really odd that this type is using `create`.  It makes so sense for it to be `ensurable`.